salt/orchestrate/services/rabbitmq.sls

```diff
@@ -2,8 +2,16 @@
  ENVIRONMENT, BUSINESS_UNIT, subnet_ids with context %}
 {% set rabbit_admin_password = salt.vault.read('secret-{}/{}/rabbitmq-admin-password'.format(BUSINESS_UNIT, ENVIRONMENT)).data.value %}
 {% if not rabbit_admin_password %}
-{% set rabbitmq_admin_password = salt.random.get_str(42) %}
-{% salt.vault.write('secret-{}/{}/rabbitmq-admin-password'.format(BUSINESS_UNIT, ENVIRONMENT), value=rabbitmq_admin_password) %}
+{% set rabbit_admin_password = salt.random.get_str(42) %}
```

Comment:
Should we just use `rabbitmq_admin_password` throughout since that's what we're calling it in vault?

Reply:
Probably not a bad idea. I'll update.